### PR TITLE
Card branch

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

**Review by sudipkundu999 — CHANGES_REQUESTED** [review]
The code looks good.
In the deploy preview, the cards look very big. Check that!
Maybe giving a height and width to the card can solve it.

File: components/card.css [hunk 1]
```diff
@@ -13,16 +12,30 @@
   width: max-content;
 }
 
-.card-image {
```
Comment (dcs-soni) [inline components/card.css:16]: You can remove this if any CSS is not applied to this class. And the code looks good to me 👍

**Review by dcs-soni — CHANGES_REQUESTED** [review]

File: components/utility.css [hunk 2]
```diff
@@ -232,6 +233,8 @@
 .d_inline {
   display: inline;
 }
+
 .d_inlineblock {
   display: inline-block;
 }
+
```
Comment (bhushannaikaroskar) [inline components/utility.css:239]: I think There should be better naming like d_inline_block so that its more readable. Rest all good.

**Review by bhushannaikaroskar — CHANGES_REQUESTED** [review]

**Review by sudipkundu999 — APPROVED** [review]
Looks good now! Good to merge!

**Review by dcs-soni — APPROVED** [review]

**Pull request merged**